spec/models/user_spec.rb

```diff
@@ -1,14 +1,15 @@
 require 'rails_helper'
 
 describe User do
-  # subject { FactoryGirl.build(:user) }
+  subject { FactoryGirl.build(:user) }
+  it { should have_many(:adventures).through(:adventure_users) }
   it { should have_many(:bucket_lists) }
 
-  # it { should have_valid(:username).when('useruser', 'iamauser12345') }
-  # it { should_not have_valid(:username).when(nil, '') }
-  #
-  # it { should validate_uniqueness_of(:username) }
-  #
-  # it { should have_valid(:email).when('user@mailhere.com', 'thisuser@mail.com') }
-  # it { should_not have_valid(:email).when(nil, '', 'user', '@', 'usermail.com', 'user@mail') }
+  it { should have_valid(:username).when('useruser', 'iamauser12345') }
+  it { should_not have_valid(:username).when(nil, '') }
+
+  it { should validate_uniqueness_of(:username) }
+  
+  it { should have_valid(:email).when('user@mailhere.com', 'thisuser@mail.com') }
+  it { should_not have_valid(:email).when(nil, '', 'user', '@', 'usermail.com', 'user@mail') }
```

Comment:
Line is too long. [94/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.